code/CSV_write.h

```diff
@@ -0,0 +1,22 @@
+#ifndef CSV_WRITE_H
+#define CSV_WRITE_H
+#include <stdint.h>
+#include "bsg.h"
+
+typedef struct {
+    uint16_t velocity[1000];
+    uint16_t angleAccPedal[1000];
+    uint16_t angleBrakePedal[1000];
+    bool requestCarStart[1000];
+    uint8_t batteryvoltage[1000];
+    uint8_t batterycurrent[1000];
+    StateBSG currentMode[1000];
+    uint16_t rpm[1000];
+    float voltage[1000];
+    float current[1000];
+}csvdata;
```

Comment:
Suggestion: you can use the other struct directly to represent a single line from CSV file as presented below

```suggestion
typedef struct {
    Vehicle vehicle;
    Engine engine;
    Battery battery;
    BSG bsg;
} VehicleData;

VehicleData csvData[10000];
```

This will help to create tests with smaller arrays instead of a fixed one with 1000 lines of data; ex:

```cpp
VehicleData shortData[100];

shorData[42].vehicle.velocity = 60;
```